lib/hb-D43J.txt

```diff
@@ -11,6 +11,13 @@ frameSetDepth_hook_addr             = 0x80070740;
 romLoadRange_hook_addr_1            = 0x8006E760;
 romLoadRange_hook_addr_2            = 0x8006E7CC;
 videoForceRetrace                   = 0x8008E120;
+simulatorRumbleStop                 = 0x80008BCC;
+mcardWrite                          = 0x800164E8;
+mcardWrite_bl_flash                 = 0x80008C38;
+mcardWrite_bl_sram                  = 0x80008CA0;
+mcardWrite_bl_eeprom                = 0x80008D3C;
```

Comment:
nit: mcardWrite_hook_addr would be more consistent (idk what to call these though). Also hooking sram only should be enough right?